src/content/docs/more-integrations/open-source-telemetry-integrations/micrometer/micrometer-metrics-registry.mdx

```diff
@@ -1,31 +1,156 @@
+These are generalized steps to set up Micrometer metrics forwarding. You may want to update code snippets as needed to fit your specific environment. 
+
+<Steps>
+    <Step>
+
+### Add OpenTelemetry Micrometer instrumentation
+
+Add OpenTelemetry Micrometer instrumentation to the alpha modules section of your `build.gradle` file:
+
+    ```java
+    //Alpha modules
+    implementation 'io.opentelemetry.instrumentation:opentelemetry-micrometer-1.5'
+    ```
+
+    </Step>
+    <Step>
+
+### Add OpenTelemetry dependencies
+        
+In the `dependencies` section, add the OpenTelemetry SDK and OTLP exporter:
+
+    ```java
+    dependencies {
+        implementation 'io.opentelemetry:opentelemetry-sdk'
+        implementation 'io.opentelemetry:opentelemetry-exporters-otlp'
+    }
+    ```
+    
+
+An example file with alpha modules and dependencies added might look something like this:
+
+```java
+plugins {
+    id 'java-library'
+    id 'org.springframework.boot'
+}
+
+bootRun {
+    mainClass.set 'io.opentelemetry.example.micrometer.Application'
+}
+
+dependencies {
+    implementation platform("io.opentelemetry:opentelemetry-bom-alpha:<JAVA_OTEL_VERSION>")
+    implementation 'io.opentelemetry:opentelemetry-sdk'
+    implementation 'io.opentelemetry:opentelemetry-exporters-otlp'
+
+    implementation platform("io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom-alpha:<OTEL_JAVA_)INSTRUMENTATION_VERSION>")
+    implementation 'io.opentelemetry.instrumentation:opentelemetry-micrometer-1.5'
+}
+```
+
+Keep in mind you will need to update the snippet with correct versioning. 
+
+    </Step>
+    <Step>
+
+### Configure Micrometer to forward data to New Relic
+
+Below is an example code snippet that directs Micrometer to use the OpenTelemetry Micrometer bridge. 
+
+This snippet updates your code so OpenTelemetry can detect Micrometer data, then forward that data to New Relic:
+
+```java
+public OpenTelemetry openTelemetry() {
+    return OpenTelemetrySdk.builder()
+        .setMeterProvider(
+            SdkMeterProvider.builder()
+                .setResource(
+                    Resource.getDefault().toBuilder()
+                        .put("service.name", "micrometer-shim")
+                        // Include instrumentation.provider=micrometer to enable micrometer metrics
+                        // experience in New Relic
+                        .put("instrumentation.provider", "micrometer")
+                        .build())
+                .registerMetricReader(
+                    PeriodicMetricReader.builder(
+                            OtlpHttpMetricExporter.builder()
+                                .setEndpoint("https://otlp.nr-data.net")
+                                .addHeader(
+                                    "api-key",
+                                    Optional.ofNullable(System.getenv("NEW_RELIC_LICENSE_KEY"))
```

Comment:
Should add a note that this depends on you having the `NEW_RELIC_LICENSE_KEY` set to your NR license key. So requires something like:
```
export NEW_RELIC_LICENSE_KEY=<INSERT_LICENSE_KEY>
```